system/control-plane/global/components/widgets/OpenTracingVisTimeline/package-lock.json

```diff
@@ -9,7 +9,7 @@
       "resolved": "https://registry.npmjs.org/@wso2-dashboards/widget/-/widget-1.2.2.tgz",
       "integrity": "sha512-j1sg6tgHonBcjeSh/DRD7nPdzEQOto18FBogbizzq2q+Y6uUx63+rqCvJl2rICQbJaxvsGLW1RrT3odDu7TZ0Q==",
       "requires": {
-        "axios": "0.17.1"
```

Comment:
These widgets are not used, can you remove the changes for the package-lock.json from widgets?